app.js

```diff
@@ -1,25 +1,104 @@
-const express = require('express')
-const logger = require('morgan')
-const cors = require('cors')
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const { connectDB } = require("./models/db");
+const bodyParser = require("body-parser");
 
-const contactsRouter = require('./routes/api/contacts')
+const {
+  listContacts,
+  getById,
+  addContact,
+  removeContact,
+  updateContact,
+  validateContact,
+} = require("./routes/api.js/contacts");
 
-const app = express()
+const app = express();
 
-const formatsLogger = app.get('env') === 'development' ? 'dev' : 'short'
+connectDB();
 
-app.use(logger(formatsLogger))
-app.use(cors())
-app.use(express.json())
+app.use(morgan("dev"));
+app.use(cors());
+app.use(bodyParser.json());
 
-app.use('/api/contacts', contactsRouter)
+app.get("/api/contacts", async (req, res) => {
+  try {
+    const contacts = await listContacts();
+    res.json(contacts);
+  } catch (error) {
+    console.error(error);
+    res.status(500).json({ message: "Internal Server Error" });
+  }
+});
```

Comment:
```suggestion
app.get("/api/contacts", async (req, res) => {
  try {
    const contacts = await listContacts();
    res.json(contacts);
  } catch (error) {
    console.error(error);
    res.status(500).json({ message: "Internal Server Error" });
  }
});
```
Te routes powinny być przekazane do folderu routes w pliku contacts.js

Reply:
I importowane tutaj 
```
app.use('/contacts', contactsRoute);
```

Reply:
Bo jak stworzyłaś folder routes to tam powinny byc routes.